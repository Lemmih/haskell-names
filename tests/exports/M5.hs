module M5 (b, c) where

data A = A { b :: A }

c = b
