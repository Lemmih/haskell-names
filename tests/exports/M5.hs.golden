Symbols (
  fromList
    [ SymValue
        { sv_origName = OrigName
                          { origPackage = Nothing
                          , origGName = GName
                                          "M5"
                                          "c"
                          }
        , sv_fixity = Nothing
        }
    , SymSelector
        { sv_origName = OrigName
                          { origPackage = Nothing
                          , origGName = GName
                                          "M5"
                                          "b"
                          }
        , sv_fixity = Nothing
        , sv_typeName = OrigName
                          { origPackage = Nothing
                          , origGName = GName
                                          "M5"
                                          "A"
                          }
        }
    ] ) (
  fromList [] )